featurepack/base-infinispan/pom.xml

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.hibernate.ogm</groupId>
+        <artifactId>hibernate-ogm-featurepack</artifactId>
+        <version>5.4.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>hibernate-ogm-featurepack-base-infinispan</artifactId>
+    <packaging>pom</packaging>
+
+    <!-- At the time of writing, Infinispan does not provide a feature pack client modules distribution -->
+    <!-- This module can be removed after solving issue: https://issues.jboss.org/browse/ISPN-8689 -->
+    <name>Hibernate OGM WildFly Feature Pack - Temporary Infinispan modules Repackaging</name>
+    <description>Temporary repackaging of Infinispan modules as a Wildfly feature pack</description>
+
+    <properties>
+        <infinispan-client-azure-version>6.1.0</infinispan-client-azure-version>
```

Comment:
This is strange name for the properties, shouldn't be something like infinispanClientAzureVersion?